packages/terra-application-docs/package.json

```diff
@@ -36,9 +36,15 @@
   },
   "dependencies": {
     "@cerner/terra-docs": "^1.0.0",
+    "@cerner/terra-dev-site": "^6.30.0",
```

Comment:
I'm rusty, will this bump with the release?

Reply:
yeah it'll get updated with the prepare for release script